cargo-nextest/src/dispatch.rs

```diff
@@ -845,9 +845,24 @@ pub struct TestRunnerOpts {
     fail_fast: bool,
 
     /// Run all tests regardless of failure
-    #[arg(long, conflicts_with = "no-run", overrides_with = "fail-fast")]
+    #[arg(
+        long,
+        name = "no-fail-fast",
+        conflicts_with = "no-run",
+        overrides_with = "fail-fast"
+    )]
     no_fail_fast: bool,
 
+    /// Number of tests that can fail before exiting test run
+    #[arg(
+        long,
+        name = "max-fail",
```

Comment:
* It would be interesting to also support a value here that's equivalent to `--no-fail-fast`, e.g. `--max-fail=all` or `--max-fail=disabled`.

* Could you make this `require_equals = true`? For newer options we've been preferring that.